bigquery_etl/cli/monitoring.py

```diff
@@ -144,6 +156,164 @@ def deploy(
     )
 
 
+def _sql_rules_from_file(custom_rules_file, project, dataset, table) -> list:
+    """Extract the SQL rules from the custom rules file."""
+    jinja_params = {
+        **{
+            "dataset_id": dataset,
+            "table_name": table,
+            "project_id": project,
+        },
+    }
+    if "format" not in jinja_params:
```

Comment:
Wouldn't this always be True given that we only define `jinja_params` right above? In which case, should we just add the key to the definition above and set the value to False there?